flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/catalog/UnknownCatalogTest.java

```diff
@@ -55,6 +56,52 @@ public class UnknownCatalogTest {
     public static final ResolvedSchema EXPECTED_SCHEMA =
             ResolvedSchema.of(Column.physical("i", INT()), Column.physical("s", STRING()));
 
+    public static final ResolvedSchema CURRENT_TIMESTAMP_EXPECTED_SCHEMA =
+            ResolvedSchema.of(Column.physical("CURRENT_TIMESTAMP", TIMESTAMP_LTZ(3).notNull()));
+
+    @Test
+    public void testUnsetCatalogWithSelect1() throws Exception {
```

Comment:
I think this does not add any test coverage, imo. There is no object resolution here. As it should be not expensive, it's up to you if you want to leave it here.

Reply:
Let's drop this

Reply:
I had added it since it is a case that I didn't see covered.  I'm fine either way and am deferring to Timo's point.